src/com/xilinx/rapidwright/router/VersalClockRouting.java

```diff
@@ -84,10 +84,9 @@ public static RouteNode routeBUFGToNearestRoutingTrack(Net clk) {
      * @param clk The current clock net to contribute routing.
      * @param startingRouteNode The intermediate start point of the clock route.
      * @param clockRegion The center clock region or the clock region that is one row above or below the center.
-     * @param adjusted A flag to guard the default functionality when routing to centroid clock region.
      * @param findCentroidHroute The flag to indicate the returned RouteNode should be HROUTE in the center or VROUTE going up or down.
      */
-    public static RouteNode routeToCentroid(Net clk, RouteNode startingRouteNode, ClockRegion clockRegion, boolean adjusted, boolean findCentroidHroute) {
+    public static RouteNode routeToCentroid(Net clk, RouteNode startingRouteNode, ClockRegion clockRegion, boolean findCentroidHroute) {
```

Comment:
`adjusted` parameter is always true -- I've constant-propagated it. Can you see a problem with that?

Reply:
Yes! I just copied it from `UltraScaleClockRouting` class when I started out, and I never set it to `false` in any calls in Versal clock routing.

Actually, you can see that `UltraScaleClockRouting` does not use a false value in any calls in symmetric routing too. It provides an overload method here https://github.com/Xilinx/RapidWright/blob/c43c9fe7e5882d1d7b49a5fb1d477cd50799af2a/src/com/xilinx/rapidwright/router/UltraScaleClockRouting.java#L91 which sets `adjusted` to false, and it is only called here: https://github.com/Xilinx/RapidWright/blob/c43c9fe7e5882d1d7b49a5fb1d477cd50799af2a/src/com/xilinx/rapidwright/examples/SLRCrosserGenerator.java#L371 for cross SLR clock routing.

Reply:
> Actually, you can see that UltraScaleClockRouting does not use a false value in any calls in symmetric routing too.

That statement is not quite accurate; I would say: `SLRCrosserGenerator` calls the `routeToCentroid(Net, RouteNode, ClockRegion)` overload, which in turn calls `routeToCentroid(Net, RouteNode, ClockRegion, boolean, boolean)` where the first boolean (`adjusted`) is set to false. So yes, in `UltraScaleClockRouting`, that `adjusted` parameter is important.